docs/list-available-operators.md

```diff
@@ -21,3 +28,49 @@ postgres-operator                  Community Operators   26m
 prometheus                         Community Operators   26m
 wildfly                            Community Operators   26m
 ```
+
+If there are no `CatalogSource` objects present in the current namespace of the command above the list will contain all Operators found in global catalogs. To include Operators available to install from namespace catalogs simply supply it with the `kubectl` command:
+
+```bash
+$ kubectl get packagemanifests
+
+NAME                               CATALOG               AGE
+...
+my-operator                        My Catalog            31m
+...
```

Comment:
You used `[...]` above to imply that additional output existed. Please chose a single format.